cmd/gateway-main.go

```diff
@@ -201,8 +201,14 @@ func gatewayMain(ctx *cli.Context) {
 	initNSLock(false) // Enable local namespace lock.
 
 	router := mux.NewRouter().SkipClean(true)
-	registerGatewayAPIRouter(router, newObject)
 
+	// Register web router when its enabled.
+	if globalIsBrowserEnabled {
+		aerr := registerWebRouter(router)
+		fatalIf(aerr, "Unable to configure web browser")
+		registerGatewayAPIRouter(router, newObject)
```

Comment:
registerGatewayAPIRouter(router, newObject) should be outside this `if` block.  Browser can be disabled.. by user.